subgraphs/notional-finance/src/mappings/notional.ts

```diff
@@ -335,3 +337,12 @@ export function handleLiquidatefCash(event: LiquidatefCashEvent): void {
 
   createLiquidate(event, currencyId, liquidator, liquidatee, amount);
 }
+
+let item: BigDecimal = BIGDECIMAL_ZERO;
+export function avg(arr: Array<BigDecimal>): BigDecimal {
```

Comment:
I don't think this function is used anymore, right?

Reply:
Was planning to remove at the end but will go ahead and remove it now.